src/regionnet.jl

```diff
@@ -34,8 +34,11 @@ else
     println("Trying to create a new region network...")
 
     # Load the network of counties
-    counties = readtable(joinpath(todata, "county-info$suffix.csv"), eltypes=[UTF8String, UTF8String, UTF8String, UTF8String, Float64, Float64, Float64, Float64, Float64, Float64, Float64])
-
+    if config["netset"] == "usa"
+    counties = readtable(joinpath(todata, "county-info.csv"), eltypes=[UTF8String, UTF8String, UTF8String, UTF8String, Float64, Float64, Float64, Float64, Float64, Float64, Float64])
+    else
+    counties = readtable(joinpath(todata, "county-info$suffix.csv"), eltypes=[UTF8String,     UTF8String, UTF8String, UTF8String, Float64, Float64, Float64, Float64, Float64,       Float64, Float64])
+    end
```

Comment:
Ah, I see.  Okay, I agree with your change.  Can you just indent the two conditional lines?